js/apps/system/_admin/aardvark/APP/frontend/js/templates/loginView.ejs

```diff
@@ -2,7 +2,7 @@
 
   <div id="loginWindow" class="login-window">
     <div class="login-logo-round">
-      <img src="img/arangodb-vertical-light.svg" />
+      <img src="img/arango-black.svg" />
```

Comment:
for some reason the login icon image is not loading:
<img width="402" height="292" alt="Image" src="https://github.com/user-attachments/assets/968e2201-6e2d-4c33-becf-1c55cc9dbd62" />

I have to check why.

Reply:
Problem found, it's because it's loading the image from the container's aardvark img folder (`http://localhost:3000/_db/_system/_admin/aardvark/img/arango-black.svg`), and the new image in the container doesn't exist, I've tried creating a container and pointing it to my local aardvark (devel branch) and now it loads:
<img width="1152" height="463" alt="loading-both" src="https://github.com/user-attachments/assets/5a92fc40-7c38-4fc3-bb89-6541ccd2c0e2" />

I understand that this problem will not occur when the container uses a new build that includes the new image, so this is really not a problem, correct me if I'm wrong, @neunhoef

Reply:
I have checked that all is well if I build a container image with oskar. I think we are good to go.